sourcegraph/sourcegraph
# SG-52032 - Add Cody AI banner to repository pages

## Description
Add Cody CTA to the repo page, and improve the existing filepage TryCodyCTA

## Ref
[SG Issue 1](https://github.com/sourcegraph/sourcegraph/issues/52032)
[SG Issue 2](https://github.com/sourcegraph/sourcegraph/issues/52034)

[Gitstart Ticket 1](https://clients.gitstart.com/sourcegraph/1/tickets/SG-52032)
[Gitstart Ticket 2](https://clients.gitstart.com/sourcegraph/1/tickets/SG-52034)

## Demo
![Screenshot 2023-05-22 at 10 13 56 AM](https://github.com/sourcegraph/sourcegraph/assets/89894075/a7e1533e-10d0-4c12-823a-f92e6bd3a43f)
![Screenshot 2023-05-22 at 10 11 32 AM](https://github.com/sourcegraph/sourcegraph/assets/89894075/1ad91e81-5035-4cea-8e65-729a662efdbc)

## Test Plan
1. Run application on this branch.
2. Go to the blob page, by clicking on a file result on the search results page
3. Compare the rendered CTA to the Figma design.
4. Go to the repository page
5. Notice that a CTA is now visible and matches the Figma design.
